packages/babel-plugin-transform-async-to-generator/test/fixtures/async-to-generator/async-arrow-in-method/actual.js

```diff
@@ -0,0 +1,11 @@
+let TestClass = {
+    name: "John Doe",
+
+    testMethodFailure()
+    {
+        return new Promise(async (resolve) => {
+            console.log(this);
+            setTimeout(resolve, 1000);
+        });
+    }
+};

```

Comment:
there's probably a good amount but we can try to autofix most of them we'll see.